cmake/configs/base.cmake

```diff
@@ -194,6 +194,12 @@ config_bool(
     DEFAULT ${default_enable_python}
 )
 
+config_bool(
```

Comment:
Why retain this?

Reply:
Still figuring out the `__ARM_FEATURE_RCPC` which is blocking removing this.